files/gus_home/config/MicrobiomeDB/model-config.xml.tmpl

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <modelConfig modelName="microbiomeModel"
              webServiceUrl="local"
-             smtpServer="127.0.0.1"
+             smtpServer="%%SMTP_HOST%%"
```

Comment:
Need to add new username/password fields here with envs and docker compose too, with empty values as defaults.